mlflow/utils/cli_args.py

```diff
@@ -121,7 +121,7 @@ def _resolve_env_manager(ctx, _, env_manager):
     default=False,
     help="If specified and there is a conda environment to be activated "
     "mlflow will be installed into the environment after it has been"
-    " activated. The version of installed mlflow will be the same as"
+    " activated. The version of installed mlflow will be the same as "
```

Comment:
```
    "mlflow will be installed into the environment after it has been "
    "activated. The version of installed mlflow will be the same as "
```

Can we also fix line 123?

Reply:
@harupy please see [f2b3ec43](https://github.com/mlflow/mlflow/pull/5761/commits/f2b3ec43445e42c5dfecf9f71282b8dd46dd9cd9)